compiler/p/codegen/OMRInstOpCodeProperties.hpp

```diff
@@ -73,7 +73,7 @@
    /* .name        = */ "add",
    /* .description =    "Add", */
    /* .opcode      = */ 0x7C000214,
-   /* .format      = */ FORMAT_RT_RA_RB,
+   /* .format      = */ FORMAT_RT_RA_RB_MEM,
```

Comment:
I don't think add has gr0 implication.

Reply:
You're right, I seem to have misread the ISA. Fixed in 8acd81952bbbc179426485bf88f69e91c28c757d.